tests/js/spec/views/settings/projectDebugFiles/customRepositories.spec.tsx

```diff
@@ -88,6 +88,11 @@ describe('Custom Repositories', function () {
     type: CustomRepoType.APP_STORE_CONNECT,
   };
 
+  beforeAll(async function () {
+    // transpile the modal upfront so the test runs fast
+    await import('sentry/components/modals/debugFileCustomRepository');
+  });
```

Comment:
What sort of speedup do we see?

Reply:
potentially 4+ seconds in the test gets moved up to this `beforeAll`. The overall test suite will still take about the same time though.

Reply:
here's a test run i ran to see how long this was taking https://github.com/getsentry/sentry/runs/5698279430?check_suite_focus=true#step:9:79